object-manipulation-1/src/get-keys.js

```diff
@@ -0,0 +1,15 @@
+/* exported getKeys */
+
+function getKeys(object) {
+
+  var propArray = [];
+  for (var key in object) {
+    propArray.push(key);
+  }
+  return propArray;
+}
+
+/* 1.) assign an empty array literal to a variable, propArray
+   2.) use a for-in loop with a variable key in object
+   3.)call the push method of the propArray array with an argument of key
```

Comment:
This is basically just code reading of your completed code. Pseudocode is meant to be a description of what is happening, not what the code is. Try revising this to describe the purpose of each line and what it is doing to help accomplish the goal.

Reply:
This seems to be a constant in all your pseudocode, try to revise your pseudocode for each of these solutions to fit this revised goal.